ninja/openapi/views.py

```diff
@@ -16,12 +16,16 @@
 __all__ = ["default_home", "openapi_json", "openapi_view", "openapi_view_cdn"]
 
 
-render_swagger = ninja_settings.DOCS_VIEW == "swagger"
-view_tpl = "ninja/swagger.html" if render_swagger else "ninja/redoc.html"
+render_doc = "swagger"
+
+if ninja_settings.DOCS_VIEW == "redoc":
```

Comment:
Can we use a walrus operator here?

Reply:
> Can we use a walrus operator here?

no - since we support pytthon3.7 and pytrhon3.6 probably works as well